### PR TITLE
EZP-29682: Cannot set default Location for objectrelationlist

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29682
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/admin/content_type/field_types.html.twig [hunk 1]
```diff
@@ -163,6 +163,7 @@
                 data-relation-root-input-selector="#{{ form.selectionDefaultLocation.vars.id }}"
                 data-relation-selected-root-name-selector="#{{ form.selectionDefaultLocation.vars.id }}-selected-root-name"
                 data-starting-location-id="{{ admin_ui_config.universalDiscoveryWidget.startingLocationId|default(1) }}"
+                data-udw-config="{{ ez_udw_config('single_container', {}) }}"
```
Comment (dew326) [inline src/bundle/Resources/views/admin/content_type/field_types.html.twig]: Why we should select only one container in `objectrelactionlist`?

Comment (ViniTou) [reply src/bundle/Resources/views/admin/content_type/field_types.html.twig]: As the descriptions says, this is "starting location for browsing for relations".

**Comment by micszo** [comment]
@ViniTou base should be changed! It occurs on 2.2.3 too.

File: src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig [hunk 2]
```diff
@@ -17,9 +17,9 @@
                 <th class="ez-relations__table-actions">
                     <button
                         {% if limit == 1 %}
-                            data-udw-config="{{ ez_udw_config('single', {}) }}"
+                            data-udw-config="{{ ez_udw_config('object_relation_single', {'starting_location_id' : default_location}) }}"
```
Comment (dew326) [inline src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig]: Isn't changing the config name a BC break?
//cc @Nattfarinn

Comment (ViniTou) [reply src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig]: @dew326 
thanks, fixed :)

**Comment by micszo** [comment]
Let's wait for Travis.

**Comment by lserwatka** [comment]
@micszo it is failing.

**Comment by lserwatka** [comment]
@micszo all green now. Can we merge it?

**Comment by micszo** [comment]
@lserwatka yes, retested and works fine.

**Comment by lserwatka** [comment]
@ViniTou please merge it up.

**Pull request merged**